README.md

```diff
@@ -1,2 +1,15 @@
 # TeamCity-TestFlight-Appium-ReactNative
 Guidance on integrating Kobiton service into the mobile app build pipeline: TeamCity, TestFlight, Appium and ReactNative.
+
+If you are using:
++ TeamCity to build app
++ TestFlight for keeping the daily app build
++ Appium to write the automation test
++ Reactnative to write app
+
+The guidelines in this repo will show you how to trigger a TeamCity build from a push on GitHub, and how to run automation tests with Kobiton.
```

Comment:
Please write something to let the readers know that Kobiton supports Appium. This doesn't highlight the importance of Kobiton in a pipeline.